<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationBarClass</class>
 <widget class="QWidget" name="NavigationBarClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NavigationBar</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btnRolls">
     <property name="maximumSize">
      <size>
       <width>85</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Rolls</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblCaption">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnPreviousRoll">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnNextRoll">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
